### PR TITLE
#78 Fixed broken relative links in the documentation

### PR DESCRIPTION
**Pull request by lucasferryinbibo**
This PR is a continuation of the following PR: https://github.com/Inbibo/adonisfx_docs/pull/77
 In this PR we have addressed:
 - Fix any relative link paths that are not coherent with the hierarchy and broken links. This has been tested on the preview tool however cross-section link references don't appear to work in the preview tool.
 - Marked Envelope attribute as animateable in AdnSkinMerge
 - Added Deformer Attributes and Envelope description to AdnSkin, AdnSimshape, AdnMuscle and AdnRibbonMuscle
 >> AdnMuscle:
image
>> AdnRibbonMuscle:
image
>> AdnSkin:
image
>> AdnSimshape:
image
>> AdnSkinMerge:
image

### CODE REVIEW

**Review by carlosmonteagudoinbibo — APPROVED** [review]
Looks good to me. Is there no more .md references or "maya/" ?

**Review by josevalverdeinbibo — APPROVED** [review]
All good for me. Just added a comment.

**Comment by lucasferryinbibo** [comment]
> Looks good to me. Is there no more .md references or "maya/" ?

`.md`. These are necessary:
image

`/maya. These are necessary:
image

File: docs/maya/muscle.md [hunk 1]
```diff
@@ -96,6 +96,11 @@ To create an AdnMuscle, follow these steps:
 | **Color**       | Color Picker |               | ✓ | Selects the line coluor from a colour wheel. Its saturation can be modified using the slider. |
 | **Fiber Scale** | Float        | 3.0           | ✓ | The scale can be modified to set a custom fiber length. |
 
+### Deformer Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Envelope**            | Float | 1.0             | ✓ | Specifies the deformation scale factor. Has a range of \[0.0, 1.0\]. Upper and lower limits are soft, values can be set in a range of \[-2.0, 2.0\]|
```
[the remark this answers is not in the document]
Comment (lucasferryinbibo) [reply docs/maya/muscle.md]: Agreed and done! We get the same output

**Review by luisalbaceteinbibo — APPROVED** [review]

**Pull request merged**